### ISSUE TITLE
Situation on external computing unclear

### ISSUE THREAD

**Issue by semeyerz** [RoboCupAtHome/RuleBook#421]
3.6.  External computing states:

Availability:
The resources must be publicly available, for use by robots of other teams,
well before and after the competition.

Which contradicts the fact, that some teams run computation on ECDs that they do not make publicly available.

The Remark of the section repeats the statement that it has to be available for other teams.

**Comment by warp1337** [comment]
Agree, this should be clarified. However, there is a technical problem. In order to isolate the teams they are usually operating in their own vlan or subnet. I guess letting everyone in every vlan or establishing another vlan for ECD's will make things even more complicated. Also, even if publ. available, I don't think people will ever use it because their would have to implement the API or interface on-site during the competition ... that's not gonna happen, ever, I guess.

So I suggest to remove the section stating the ECD's must be publ. available since it is really not feasible. I, personally, would also not use the ECD of another team because I can also not make sure
that it will be up and running when I need it for testing/during the task. Dont get me wrong here, I would love to use other teams' cool stuff, but I think it is just not feasible and needs clarification in the rulebook.

When it comes to external cloud services, I fully agree. This is a totally different story since they are on the internet and everybody should be able to access them from their vlan/subnet

**Comment by kyordhel** [comment]
Directly related, #432 points to the removal of all cloud computing resources from the competition. Therefore, if you are running an amazing webservice, other teams should be able to access it. To me, having teams running magical blackboxes with closed-sourced code will lead to nowhere but I get the point.

@semeyerz please make a PR removing all access restrictions for external computing resources.

**Comment by kyordhel** [comment]
Directly related, #432 points to the removal of all cloud computing resources from the competition. Therefore, if you are running an amazing webservice, other teams should be able to access it. To me, having teams running magical blackboxes with closed-sourced code will lead to nowhere but I get the point.

@semeyerz please make a PR removing all access restrictions for external computing resources.
